moit-api/src/main/kotlin/com/mashup/moit/controller/user/AuthController.kt

```diff
@@ -48,7 +52,8 @@ class AuthController(
     @Operation(summary = "회원가입", description = "회원가입 API")
     @PostMapping("/sign-up")
     fun signUp(@RequestBody request: UserRegisterRequest): ResponseEntity<Any> {
-        val jwtToken = userFacade.createUser(request).let { jwtTokenSupporter.createToken(UserInfo.from(it)) }
+        val jwtToken = userFacade.createUser(request)
+            .let { jwtTokenSupporter.createToken(UserInfo.from(it)) }
```

Comment:
```suggestion
        val jwtToken = jwtTokenSupporter.createToken(UserInfo.from(userFacade.createUser(request)))
```
약간 스타일 차이인데 저는 이런 인스턴스 생성 부분에서는 scope function을 쓰지 않고 인스턴스 생성 로직 (jwtTokenSupporter.createToken~~) 부분을 가장 바깥쪽으로 빼는게 예쁜것 같아요!

Reply:
맞네요. 더 이해하기 좋을 것 같습니다 👍🏻